src/styles/blocks/icons.scss

```diff
@@ -0,0 +1,33 @@
+
+.icon {
+  display: flex;
+  vertical-align: middle;
```

Comment:
Replace 'vertical-align: middle;' with 'align-items: center;' to properly center the icon within the flex container.